src/content/blog/future-of-astro-zero-js-view-transitions.mdx

```diff
@@ -0,0 +1,106 @@
+
+## Upgrade
+
+To switch to the native new View Transition API, it’s as simple as adding the `@view-transition` at-rule to your website. Here’s all the CSS you need:
+
+```css
+@view-transition {
+  navigation: auto;
+}
+```
+
+To enable cross-document view transitions, both "ends" of the transition must include this rule. A shared page layout component can be a good place to add this to your codebase.
+
+If you were already using Astro View Transitions powered by our `<ViewTransitions />` component, then you are now safe to remove that component as well. This will remove the injected backwards-compatible JavaScript router from your page, reducing the overall complexity of your client-side code.
+
+
+```diff
+---
+- import { ViewTransitions } from "astro:transitions";
+---
+<head>
+-  <ViewTransitions fallback="none" />
++  <style>
++    @view-transition {
++      navigation: auto;
++    }
++  <style>
+</head>
+```
+
+If you would still like your view transitions to run in older browsers, you are also welcome to keep using our backwards-compatible `<ViewTransitions />` component. You can migrate whenever you are ready. We have no plans to remove support for the `<ViewTransitions />` component at this time.
+
+## What about transition:persist?
+
+[Persistent element transitions](https://docs.astro.build/en/guides/view-transitions/#maintaining-state) became a popular feature in Astro. The `transition:persist` directive told Astro to persist an element across navigation without interrupting or disrupting it's state. This enabled UI like audio and video players to continue to play across pages, like in <a href="https://twitter.com/charca">Maxi Ferreira's</a> <a href="https://astro-records.pages.dev/">music player demo</a>.
+
+Unfortunately, this feature is not yet supported by native cross-document navigation in the browser. Native support for persistent audio and video players may be coming  in the future, but in the meantime you can continue to use our `<ViewTransitions />` component if you need this additional functionality.
+
+### The Future of Astro
+
+We believe this is one of the most significant advancements to come out of Chromium and the browser spec process in recent memory. Following links across websites is a fundamental behavior of the web, and this represents the largest meaningful change in recent memory to how that behavior works.
+
+Developers can take advantage of this feature today or wait as it rolls out to more users. Whenever you're ready, it’s essentially a one-line change of CSS that requires no architectural overhaul to your existing Astro code to enable. To get started, you can learn more about [Astro View Transitions](https://docs.astro.build/en/guides/view-transitions). We will be updating our documentation over the next few weeks to point developers to these [new native browser APIs](https://developer.chrome.com/docs/web-platform/view-transitions).
```

Comment:
```suggestion
Developers can take advantage of this feature today or wait as it rolls out to more users. Whenever you're ready, it’s essentially a one-line addition of CSS that requires no architectural overhaul to your existing Astro code to enable. To get started, you can learn more about [Astro View Transitions](https://docs.astro.build/en/guides/view-transitions). We will be updating our documentation over the next few weeks to include using these [new native browser APIs](https://developer.chrome.com/docs/web-platform/view-transitions).
```
I do like "addition" vs change because it's very likely they don't have anything like this in their code already?

"pointing them to" some external resources sounds a bit like "go away, you're their problem now, we absolve ourselves". So, a more gentle, assuring edit perhaps?

Also, this does negate any effort in removing the previous VT components they may have around their project, so just making you aware of that. (I would agree, though, that that doesn't represent an *architectural* change... just some tidying/housekeeping. BUT, there may still be some for whom stripping out their existing components feels like not *zero* work.)